tests/cpu/CPUProcessor_tests.cpp

```diff
@@ -814,6 +814,10 @@ OCIO_ADD_TEST(CPUProcessor, with_several_ops)
                                                         &resImg[0],  OCIO::CHANNEL_ORDERING_RGBA,
                                                         NB_PIXELS, 1e-7f);
 
+#if 0
```

Comment:
The code below the #if will not be executed ever. I assume this is not intended?

Reply:
Agreed, this test is overly strict since the hash value may be changed by insignificant differences in LUT values due to rounding differences.  The key is that it passes the ComputeValues test, which apparently it does.  

Per the comment on line 822, it could be replaced by a check that the cpuProcessor contains only a single Lut1D.  Perhaps a simple way to do that would be do test the length of the string and test that it contains "<Lut1D".

Reply:
yes, that sounds good. I'll give that a go